packages/cf-util-http/src/http.js

```diff
@@ -184,10 +184,7 @@ export function request(method, url, opts, callback) {
 
         return response.text().then(text => {
           if (callback) {
-            callback(
-              undefined,
-              wrapResponse(headers, status, text, response)
-            );
+            callback(undefined, wrapResponse(headers, status, text, response));
```

Comment:
Seems to fit. The horizontal line is the 80 char limit.

![image](https://cloud.githubusercontent.com/assets/318208/26628556/71890e9a-45f6-11e7-8202-ef78c9fba2da.png)